### PR TITLE
Hot fix loreco

### PR DESCRIPTION
**Pull request by dcolleoni**

### CODE REVIEW

File: ifrs17/Test/TechnicalMarginTest.ipynb [hunk 1]
```diff
@@ -315,6 +446,17 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Gross : multiple switch",
+                "\n",
+                "\nSwitch in the in force compenent, opposite switch in the new business component. CL AoC Step to correct the allocation of the combined section."
```
Comment (nnikolopoulos) [inline ifrs17/Test/TechnicalMarginTest.ipynb:454]: typo: compenent -> component.

File: ifrs17/Test/TechnicalMarginTest.ipynb [hunk 2]
```diff
@@ -364,10 +506,278 @@
...
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Reinsurance with Gross with CL balancing item"
```
Comment (nnikolopoulos) [inline ifrs17/Test/TechnicalMarginTest.ipynb:718]: Should it be "LC balancing item" instead?

File: ifrs17/Test/TechnicalMarginTest.ipynb [hunk 3]
```diff
@@ -176,29 +173,145 @@
...
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Test runner ",
+                "\nprovided inputs and benchmarks computes the results and calls the error logger routine."
```
Comment (nnikolopoulos) [inline ifrs17/Test/TechnicalMarginTest.ipynb:241]: Capitalize provided.

**Review by nnikolopoulos — APPROVED** [review]

File: ifrs17/Test/TechnicalMarginTest.ipynb (hunk 2 above)
Comment (dcolleoni) [reply ifrs17/Test/TechnicalMarginTest.ipynb:718]: it's actually CL: combined liability AoC Step

**Pull request merged**